test/functionalTest/cases/0981_POST_v2_attributes/append_and_append_only_combinations.test

```diff
@@ -146,7 +146,7 @@ Date: REGEX(.*)
 
 03. POST /v2/entities/E1 attribute 'attr2'
 ==========================================
-HTTP/1.1 201 Created
+HTTP/1.1 204 No Content
 Content-Length: 0
 Location: /v2/entities/E1
```

Comment:
Here it is.
Do we want Location as response HTTP header for modifications of an entity?